sites/default/sqlconf.php

```diff
@@ -1,30 +1,31 @@
 <?php
 //  OpenEMR
```

Comment:
@stu01509 , will need to revert the changes in this script (note we have all done this at one time or another :) )

Reply:
Thanks for your mentioned, I removed it.